src/vs/workbench/contrib/chat/browser/chatQuick.ts

```diff
@@ -173,14 +175,24 @@ class QuickChat extends Disposable {
 			});
 		}
 
-		this.registerListeners();
+		this.sash?.dispose();
+		this.sash = this._register(new Sash(parent, { getHorizontalSashTop: () => parent.offsetHeight }, { orientation: Orientation.HORIZONTAL }));
```

Comment:
`sash: MutableDisposable<Sash>` is a nice pattern for this. But shouldn't you be caching it across renders, same with `this.widget`?

Reply:
Addressed in https://github.com/microsoft/vscode/pull/191431